domain/src/main/java/hous/release/domain/usecase/PostSignUpUseCase.kt

```diff
@@ -0,0 +1,17 @@
+package hous.release.domain.usecase
+
+import hous.release.domain.repository.AuthRepository
+import javax.inject.Inject
+
+class PostSignUpUseCase @Inject constructor(
+    private val authRepository: AuthRepository
+) {
+    suspend operator fun invoke(
+        birthday: String,
+        fcmToken: String,
+        isPublic: Boolean,
+        nickname: String,
+        socialType: String,
+        token: String
+    ) = authRepository.postSignUp(birthday, fcmToken, isPublic, nickname, socialType, token)
```

Comment:
```suggestion
    suspend operator fun invoke(
        birthday: String,
        isPublic: Boolean,
        nickname: String,
    ) = authRepository.postSignUp(birthday, fcmToken, isPublic, nickname, socialType, token)
```  
이 부분 `""` 값 이제 넣어주지 않아도 돼서 빼주면 좋을 것 같습니당 ~ 뷰모델에서도~